pkg/oam/util/helper.go

```diff
@@ -226,7 +226,13 @@ func PassLabelAndAnnotation(parentObj oam.Object, childObj labelAnnotationObject
 
 // GetCRDName return the CRD name of any resources
 // the format of the CRD of a resource is <kind purals>.<group>
+// Now the CRD name of a resource could also be defined as `definition.oam.dev/name` in `metadata.annotations`
 func GetCRDName(u *unstructured.Unstructured) string {
```

Comment:
Please also refactor the function Name, it should not be `GetCRDName ` , name it `GetDefinitionName` seems better